wiki/Tournaments/Official_support/en.md

```diff
@@ -0,0 +1,169 @@
+# Official tournament support
+
+The [osu!team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments that abide by a certain set of criteria and expectations with extra support, at their discretion.
```

Comment:
```suggestion
The [osu!team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments, which abide by a certain set of criteria and expectations, with extra support, at their discretion.
```
may read better this way i think?

Reply:
forgot to comment on this but i think your change reads worse, the pauses (commas) sound unnatural to me when read aloud